kelo_hitbot_driver/MotionCommand command1
kelo_hitbot_driver/MotionCommand command2
int32 ovl
float32 blendRadius
uint8 offsetFlag
kelo_hitbot_driver/Axes offset

